backend/Dockerfile.db-rollback

```diff
@@ -1,4 +1,4 @@
-FROM gradle:6.9.1-jdk11-hotspot AS build
+FROM gradle:7.2.0-jdk11-hotspot AS build
```

Comment:
this also needs to be 7.5

Reply:
I pushed a commit correcting the version.